Common/src/interpolation_structure.cpp

```diff
@@ -2719,3 +2729,1240 @@ bool CSlidingMesh::CheckPointInsideTriangle(su2double* Point, su2double* T1, su2
 
   return (check == 3);     
 }
+
+/*--- Radial Basis Function Interpolator ---*/
+CRadialBasisFunction::CRadialBasisFunction(void):  CInterpolator() { }
+
+CRadialBasisFunction::CRadialBasisFunction(CGeometry ****geometry_container, CConfig **config,  unsigned int iZone, unsigned int jZone) :  CInterpolator(geometry_container, config, iZone, jZone) {
+
+  /*--- Initialize transfer coefficients between the zones ---*/
+  Set_TransferCoeff(config);
+
+}
+
+CRadialBasisFunction::~CRadialBasisFunction() {}
+
+void CRadialBasisFunction::Set_TransferCoeff(CConfig **config) {
+
+  int rank=MASTER_NODE, nProcessor=SINGLE_NODE;
```

Comment:
rank and size are already member variables of the base class and properly initialized. So you can remove them here.